dockeradapter/filterdockerstream.go

```diff
@@ -24,8 +25,14 @@ const (
 	stdWriterSizeIndex = 4
 )
 
+// frameHeader holds the steam type and payload size for a Docker stream frame
```

Comment:
oh, no, it was just a trivial s/steam/stream in the comment, sorry should've been more clear